shared/common-adapters/floating-menu/menu-layout/index.native.js

```diff
@@ -31,12 +31,14 @@ const MenuRow = (props: MenuRowProps) => (
     {props.view || (
       <>
         <Box2 direction="horizontal" fullWidth={true} centerChildren={true}>
+          {props.decoration && <Box style={styles.flexOne} />}
```

Comment:
this is a nice solution for having a centered item and something on the side without having to set `width`s. A box with `flex: 1` on both sides will make sure the center item stays centered.